docs/lib/snippets/_shared/todo_tables.drift.dart

```diff
@@ -459,6 +475,14 @@ class $$TodoItemsTableOrderingComposer
   }
 }
 
+class $$TodoItemsTableWithReferences {
+  // ignore: unused_field
```

Comment:
Do we really still need this with the `ignore_for_file: type=lint` at the top?

Reply:
Done